Jenkinsfile

```diff
@@ -374,7 +374,6 @@ def releasePackages(){
     python setup.py sdist
     echo "Uploading to ${REPO_URL} with user \${TWINE_USER}"
     python -m twine upload --username "\${TWINE_USER}" --password "\${TWINE_PASSWORD}" --skip-existing --repository-url \${REPO_URL} dist/*.tar.gz
-    python -m twine upload --username "\${TWINE_USER}" --password "\${TWINE_PASSWORD}" --skip-existing --repository-url \${REPO_URL} wheelhouse/*.whl
```

Comment:
We still produce one .whl file (the universal one), but it's in `dist/`, not in `wheelhouse/`

Reply:
That one only matches `*.tar.gz`, doesn't it?

Reply:
Oh, right. So did it not get uploaded at all for 6.13? PyPI shows ok but I didn't look closely.

Reply:
I guess it only uploaded the tar.gz package, which is a bit slower to install than the wheel. Not sure if this warrants a 6.13.1, though.